src/Data/Context.cs

```diff
@@ -9,5 +10,20 @@ public class Context : DbContext
         public Context(DbContextOptions<Context> options) : base(options)
         {
         }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<EDetection>().HasData(new EDetection[]
+            {
+                new EDetection { Id = 1, Class = "pet", Score = 1, Timestamp = DateTime.UtcNow },
+                new EDetection { Id = 2, Class = "pet", Score = 5, Timestamp = DateTime.UtcNow },
+                new EDetection { Id = 3, Class = "pet", Score = 8, Timestamp = DateTime.UtcNow },
+                new EDetection { Id = 4, Class = "pet", Score = 4, Timestamp = DateTime.UtcNow },
+                new EDetection { Id = 5, Class = "pet", Score = 3, Timestamp = DateTime.UtcNow },
+                new EDetection { Id = 6, Class = "pet", Score = 9, Timestamp = DateTime.UtcNow },
+                new EDetection { Id = 7, Class = "pet", Score = 1, Timestamp = DateTime.UtcNow },
+                new EDetection { Id = 8, Class = "pet", Score = 1, Timestamp = DateTime.UtcNow },
+            });
+        }
```

Comment:
Are those changes really needed? Can't we just insert stuff inside our created migration without touching the context?

Reply:
Should we also add some more data and some different classes? Won't we need more of them on frontend side?